pyroengine/pi_utils/.env

```diff
@@ -0,0 +1,3 @@
+ROOTER_LOGIN = "ROOTER_LOGIN"
+ROOTER_PASSWORD = "ROOTER_PASSWORD"
+ROOTER_IP = "ROOTER_IP"
```

Comment:
missing EOF

Reply:
And I would argue that this should be put in gitignore instead and we should add instructions to specify where this information should be added :thinking: